packages/ibm-products/src/components/SidePanel/SidePanel.tsx

```diff
@@ -896,13 +902,13 @@ export let SidePanel = React.forwardRef(
     return shouldRender ? (
       <>
         {/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions */}
-        <div
+        <Section
           {...getDevtoolsProps(componentName)}
           {...rest}
           id={id}
           className={mainPanelClassNames}
           ref={sidePanelRef}
-          role="complementary"
+          as="aside"
```

Comment:
While `as="aside"` is a better implementation than `role="complementary"`, we likely should support the side panel as a `dialog` when using the slide-over variant.

That said, this matches our current implementation so we can open a separate issue for adding support to use the side panel as a dialog.

Reply:
I guess that makes sense... actually I thought I saw a ticket about that yesterday but now I can't find it.